php/src/Webhook.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Svix;
+
+class Webhook
+{
+    public function __construct(string $secret)
+    {
+        $this->secret = base64_decode($secret);
+    }
+
+    public function verify($payload, $headers)
+    {
+        if (!isset($headers['svix-id']) || !isset($headers['svix-timestamp']) || !isset($headers['svix-signature'])) {
+            throw new Exception\WebhookVerificationException("Missing required headers");
+        }
+
+        $msgId = $headers['svix-id'];
+        $timestamp = $headers['svix-timestamp'];
+        $msgSignature = $headers['svix-signature'];
+
+        $toSign = "{$msgId}.{$timestamp}.{$payload}";
+        $signature = self::sign($this->secret, $toSign);
+
+        $passedSignatures = explode(' ', $msgSignature);
+        foreach ($passedSignatures as $versionedSignature) {
+            $sigParts = explode(',', $versionedSignature, 2);
+            $version = $sigParts[0];
+            $expectedSignature = $sigParts[1];
+
+            if (strcmp($version, "v1") != 0) {
+                continue;
+            }
+
+            if (strcmp($signature, $expectedSignature) == 0) {
```

Comment:
Not the end of the world in this case, but to prevent timing attacks you want to use a constant time comparison function.

I tried to look up an example online, and I found this: https://github.com/stripe/stripe-php/blob/f6724447481f6fb8c2e714165e092adad9ca470a/lib/WebhookSignature.php#L51

Reply:
looks like `hash_equals` is available in php 5.6+ (which I currently have this lib locked to) so we should be good using that (just updated) unless you think we need to widen our support window.

Reply:
Support Windows? The OS? Why, is newer PHP not available there?

Reply:
Sorry by `widen our support window` I meant support PHP lower than 5.6 =)

Reply:
I didn't see the `widen our`. Sorry, I'm a skimmer, not a reader. :)

No need, it's god the way it is.